build-info-extractor/src/test/java/org/jfrog/build/extractor/clientConfiguration/util/spec/SpecsHelperIntegrationTest.java

```diff
@@ -44,6 +46,47 @@ protected void cleanup() throws IOException {
         deleteContentFromRepo(localRepo);
     }
 
+    @Test
+    public void propsTest() throws Exception {
```

Comment:
I like this test method, but I think that it should not be part of this class.
This class uses the "test cases" infrastructure, which executes multiple tests (see the *integrationTests* method in the same class).
So there are probably two options. Move this method into a new class (see the *DownloadTest* class for example) or replace this test with additional test cases.